src/hooks/useCampPartygoers.ts

```diff
@@ -15,6 +15,7 @@ import { useSelector } from "./useSelector";
  * @param venueName venueName to filter partygoers by
  */
 export const useCampPartygoers = (venueName: string): WithId<User>[] => {
+  // TODO: rename this something more meaningful. venue/room/etc Partygoers? What does it actually check?
```

Comment:
- [ ] [TODO] rename this something more meaningful. "venue/room/etc Partygoers"? What does it actually check? The word `Camp` is meaningless here now that the old template is gone.

Reply:
Maybe [OOS/Future]?

Reply:
resolved in https://github.com/sparkletown/sparkle/pull/840/commits/90db5965ac8dd6cb7de7b26239f27ee379370883